scripts/wrong-filename.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: MIT
+
+import sys
+from pathlib import Path
+import re
+
+OUTPUT_FILE = Path("inconsistent-filenames.txt")
+
+IGNORE_SET = {
+    ">",
+    "<",
+    "<>",
+    ":",
+    "?",
+    "|",
+    "jc.json",
+    "lid.libuser",
+    "mc.cli",
+    "mc.fm",
+    "pacman d",
+    "pacman f",
+    "pacman q",
+    "pacman r",
+    "pacman s",
+    "pacman t",
+    "pacman u",
+    "parted",
+    "print.runmailcap",
+    "print.win",
+    "print.zsh",
+    "python m json.tool",
+    "rename",
+    "snap.esa",
+    "snap.pkg",
+}
+
+
+def normalize(text: str) -> str:
+    """
+    Normalize a string:
+    - replace '-' with spaces
+    - lowercase
+    - collapse multiple spaces into one
+    - strip leading/trailing whitespace
+    """
+    text = text.replace("-", " ").lower().strip()
+    text = re.sub(r"\s+", " ", text)
+    return text
+
+
+def check_file(path: Path) -> str | None:
+    """Check a single markdown file for name/title consistency."""
+    filename = path.name
+
+    # Remove known suffixes
+    command_file = filename
+    for suffix in (".md", ".fish", ".js", ".1", ".2", ".3"):
```

Comment:
You should make it so that if the filename starts with the same letters as the title + a dot, it would disregard the rest. That would reduce the maintenance burden